apps/spruce/src/pages/projectSettings/tabs/GithubAppSettingsTab/transformers.ts

```diff
@@ -1,20 +1,60 @@
+import {
+  adhocRequester,
+  githubMergeRequester,
+  githubPRRequester,
+  gitTagRequester,
+  gitterRequester,
+  patchRequester,
+  triggerRequester,
+} from "constants/requesters";
 import { ProjectSettingsTabRoutes } from "constants/routes";
 import { ProjectSettingsQuery } from "gql/generated/types";
+import { StringMap } from "types/utils";
 import { FormToGqlFunction, GqlToFormFunction } from "../types";
 
 type Tab = ProjectSettingsTabRoutes.GithubAppSettings;
 
+// Maintain an array to guarantee a consistent order of the requesters on the page.
+const requestersArray = [
```

Comment:
You could also try RJSF's [order property](https://rjsf-team.github.io/react-jsonschema-form/docs/version-4.2.3/usage/objects#specifying-property-order) if you like

Reply:
using enum values solved my problem here :)